core/DataAccess/Model.php

```diff
@@ -333,6 +333,18 @@ public function getSitesWithInvalidatedArchive($numericTable)
         return $result;
     }
 
+    /**
+     * Returns whether there is at least one visit that ended after the given timestamp.
+     *
+     * @param $tsArchived
+     * @return bool
+     */
+    public function hasEncounteredVisitsSinceLastArchive($tsArchived)
+    {
+        $rows = Db::fetchAll("SELECT idvisit FROM " . Common::prefixTable('log_visit') . " WHERE visit_last_action_time >= ? LIMIT 1", [$tsArchived]);
```

Comment:
I'm a bit worried this will do a full table scan. Can we include the idSite? This will be likely needed anyway I suppose to check if there were visits for that specific site